doc/DeveloperGuide.md

```diff
@@ -60,7 +60,16 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
 `* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
+`* *` | user | edit my details | easily change my details if I input it incorrectly
+`* *` | user | undo my previous input |
+`* *` | user | redo my previous undo |
 `*` | user with many persons in the address book | sort persons by name | locate a person easily
+`*` | user with many persons in the address book | sort persons by tag | locate tags easily
```

Comment:
Do you mean `locate a person based on sorted tags`? If so, then I think `find persons by tag` is better than sort.